webpack.config.js

```diff
@@ -28,16 +28,27 @@ module.exports = {
         test: /\.css$/,
         use: [MiniCssExtractPlugin.loader, "css-loader", "postcss-loader"],
       },
+      {
+        test: /\.svg$/,
+        use: [
+          {
+            loader: "svg-url-loader",
+            options: {
+              limit: 10000,
+            },
+          },
+        ],
+      },
     ],
   },
   plugins: [
     new MiniCssExtractPlugin({
       filename: "styles.css",
       chunkFilename: "styles.css",
     }),
-    new HtmlWebPackPlugin({
+    new HtmlPlugin({
       template: "./public/index.html",
-      filename: "./index.html",
+      favicon: "./public/icons/favicon.ico",
```

Comment:
Added favicon here. Also removed `filename` because it defaults to index.html already